yb-voyager/src/utils/commonVariables.go

```diff
@@ -124,9 +135,10 @@ type TableColumnsDataTypes struct {
 }
 
 type UnsupportedQueryConstruct struct {
-	ConstructTypeName string
-	Query             string
-	DocsLink          string
+	ConstructTypeName      string
+	Query                  string
+	DocsLink               string
+	MinimumVersionsFixedIn map[string]*version.YBVersion // key: series (2024.1, 2.21, etc)
```

Comment:
we don't need `isFixedIn` here as we will get the filtered ones from `queryissue` pkg, right?

Reply:
We still need because we need to include in the report the (higher) versions that it might be supported in. So, for both UnsupportedFeature, UnsupportedQueryConstruct, we'd need to store `MinimumVersionsFixedIn`.

Reply:
yeah, for map I got it, I was asking about the `IsFixedIn()` function for the struct, to check the construct is fixed or not.

Reply:
I added it based on our previous discussion:
Ideally all filtering of issues should happen in queryissue pkg layer, so we should not need to filter here. But until all code is moved out, we should also filter in analyze-schema layer. 

Although now you're moving a majority of the code from analyze-schema into queryissues, there will still be some regex-based code in analyze-schema layer, I assume?

Reply:
> Although now you're moving a majority of the code from analyze-schema into queryissues, there will still be some regex-based code in analyze-schema layer, I assume?

yeah right